x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -520,7 +562,11 @@ export class RemoteClusterForm extends Component {
           )}
           color="danger"
           iconType="cross"
-        />
+        >
+          <EuiDelayRender>
+            {messagesToBeRendered}
+          </EuiDelayRender>
+        </EuiCallOut>
```

Comment:
Can we render the messages outside of the callout? This prevents the callout from rendering with extra empty space inside of it (since the component renders an empty element when children are provided).

```jsx
      <Fragment>
        <EuiSpacer size="m" />
        <EuiCallOut
          data-test-subj="remoteClusterFormGlobalError"
          title={(
            <FormattedMessage
              id="xpack.remoteClusters.remoteClusterForm.errorTitle"
              defaultMessage="Fix errors before continuing."
            />
          )}
          color="danger"
          iconType="cross"
        >
        </EuiCallOut>
        <EuiDelayRender>
          {messagesToBeRendered}
        </EuiDelayRender>
      </Fragment>
```